pocketpy/PSPBUILD

```diff
@@ -0,0 +1,27 @@
+pkgname=pocketpy
+pkgver=1.0.9
+pkgrel=1
+pkgdesc="Python interpreter for game scripting"
+arch=('mips')
+license=('Apache-2.0 license')
+url="https://github.com/blueloveTH/pocketpy"
+makedepends=()
+source=("git+https://github.com/blueloveTH/pocketpy#commit=9d91801626f6ba731a4d0108cfcc6d3b8b518797")
+sha256sums=('SKIP')
+
+build() {
+    cd $pkgname
+    mkdir -p build
+    cd build
+    cmake -Wno-dev -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=$PSPDEV/psp/share/pspdev.cmake -DPK_BUILD_STATIC_LIB=ON ..
+    make --quiet
+}
+
+package() {
+    cd "$pkgname/build"
+
+    mkdir -m 755 -p "$pkgdir/psp/lib" "$pkgdir/psp/include" "$pkgdir/psp/include/pocketpy"
+    install -m 644 libpocketpy.a "$pkgdir/psp/lib"
+    install -m 644 ../include/pocketpy.h "$pkgdir/psp/include"
+    install -m 644 ../include/pocketpy/*.h "$pkgdir/psp/include/pocketpy"
```

Comment:
Can you also install the license into the package?

Reply:
Yeah -- I'll get that by the end of the day.